bin/notificators/hard_threshold_notificator.py

```diff
@@ -45,11 +45,13 @@ def send_slack_notification(self, query, is_send_to_user=True, is_send_to_admin_
             user=query.executing_user
         )
 
-        text = self.config.SLACK_HARD_THRESHOLD_MESSAGE.format(**params)
+        text_1 = self.config.SLACK_HARD_THRESHOLD_MESSAGE_1.format(**params)
+        text_2 = self.config.SLACK_HARD_THRESHOLD_MESSAGE_2.format(**params)
+        text_3 = self.config.SLACK_HARD_THRESHOLD_MESSAGE_3.format(**params)
```

Comment:
🔴 
Please name all the variables according to their function.
Imagine you're interested in using athena_alerter in a different company. 

How would you react if you were asked to set `SLACK_HARD_THRESHOLD_MESSAGE_2`? What does it do :)?

---
I'd also suggest to keep previous name `SLACK_HARD_THRESHOLD_MESSAGE` unchanged, as not to introduce a breaking change for `athena_alerters` users (unlikely, but as a practice) without a major reason.

Reply:
👍

Reply:
I went back to the original variable name. I have marked additional texts as "additional"

Reply:
🔴 
Variables names are still not descriptive.

`SLACK_HARD_THRESHOLD_MESSAGE_ADDITIONAL_PRIVATE`
`SLACK_HARD_THRESHOLD_MESSAGE_ADDITIONAL_ADMIN_CHANNEL`

Same for: `text_additional_1`, `text_additional_2`

Reply:
Done 👍